application-ckeditor-ui/src/main/resources/CKEditor/ImageSelectorService.xml

```diff
@@ -47,57 +47,33 @@
 
 {{velocity}}
 #if ($xcontext.action == 'get')
+#set ($selectorUIXs = $services.uix.getExtensions('org.xwiki.contrib.ckeditor.pluginsImageSelector', {'sortByParameter' : 'priority'}))
```

Comment:
I know the group id is currently ``org.xwiki.contrib`` but that's a mistake. It should have been ``org.xwiki.contrib.ckeditor``. So I'd rather use ``org.xwiki.contrib.ckeditor.plugins.imageSelector``.